### PR TITLE
Add error message when sending friend request to friend

### LINKED ISSUE #279: Change error message from "Something went to wrong" to tell the user they already have that friend added
Issue: When you try adding a friend that you already have you get an error message that says "something went wrong" we should change it to something like "You are already friends" 

**Screen recording:**

https://user-images.githubusercontent.com/93608357/200979713-73229211-6dd8-4e9b-b635-32a3c10b6158.mov

### PR DESCRIPTION
**Pull request by hyj1204**
### What this PR does 📖
- Add error message for existed friend.
- Move the error message below the text field.
- Fix the wrong error message mentioned in this PR #276

https://user-images.githubusercontent.com/14248245/201402402-afaf1585-fc64-42c5-9620-e2b5c6cfc9cc.mov

### Which issue(s) this PR fixes 🔨
- Resolve #279

### CODE REVIEW

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
Worked with trying to add myself and add someone I was already friends with, awesome job! 🔥 🎉

**Review by dariusc93 — APPROVED** [review]

File: src/components/main/friends/sidebar/mod.rs [hunk 1]
```diff
@@ -166,7 +166,7 @@ pub fn FindFriends(cx: Scope, account: Account, add_error: UseState<String>) ->
                     remote_friend.set(evt.value.clone());
                 },
                 on_enter: move |_| {
-                    let did = DID::try_from(String::from("did:key:") + &(remote_friend.clone().to_string()));    
+                    let did = DID::try_from(String::from("did:key:") + &(remote_friend.clone().to_string()));
```
Comment (InfamousVague) [inline src/components/main/friends/sidebar/mod.rs]: small change, should we use the`format!("did:key:{}", remote_friend.clone())` macro instead?

File: src/components/main/friends/sidebar/mod.rs [hunk 2]
```diff
@@ -201,8 +200,8 @@ pub fn FindFriends(cx: Scope, account: Account, add_error: UseState<String>) ->
                 icon: Shape::Plus,
                 on_pressed: move |e: UiEvent<MouseData>| {
                     e.cancel_bubble();
-    
-                    let did = DID::try_from(String::from("did:key:") + &(remote_friend.clone().to_string()));    
+
+                    let did = DID::try_from(String::from("did:key:") + &(remote_friend.clone().to_string()));
```
Comment (InfamousVague) [inline src/components/main/friends/sidebar/mod.rs]: same as above

**Review by InfamousVague — CHANGES_REQUESTED** [review]

**Review by lgmarchi — APPROVED** [review]
Good PR, improve feedback for the user. 

If it is possible, declare did variable just one time in this file. Approved.

File: src/components/main/friends/sidebar/mod.rs (hunk 2 above)
Comment (hyj1204) [reply src/components/main/friends/sidebar/mod.rs]: Updated

File: src/components/main/friends/sidebar/mod.rs (hunk 1 above)
Comment (hyj1204) [reply src/components/main/friends/sidebar/mod.rs]: Updated

**Review by InfamousVague — APPROVED** [review]

**Pull request merged**